src/documentation/docs/Sections/ProductDetailSection.mdx

```diff
@@ -16,29 +18,34 @@ import PropsTableRow from "@/components/internal/Documentation/PropsTable/compon
   <PropsTableRow value="headline" type="string" required>
     A headline for the ProductDetailSection.
   </PropsTableRow>
-  <PropsTableRow value="buttonText" type="string" optional>
-    Text to display on a clickable button. The button is only rendered, when `buttonText` is provided.
-  </PropsTableRow>
-  <PropsTableRow value="propertyList" type="ProductProperty[]" required>
-    List of product property lists that will be rendered into bullet lists under the price. Each propertyList entry consists of <span class="code">title: string, properties[]</span>, where a property again consists of <span class="code">id: string, name: string</span> - and results in a new bullet list with a custom title.
-  </PropsTableRow>
   <PropsTableRow value="description" type="string" required>
   Description of the product that will be displayed under the headline.
   </PropsTableRow>
   <PropsTableRow value="price" type="string" required>
   Price of the product that will be displayed under the description.
   </PropsTableRow>
+  <PropsTableRow value="buttonText" type="string">
+    Text to display on a clickable button. The button is only rendered, when `buttonText` is provided.
+  </PropsTableRow>
+  <PropsTableRow value="propertyList" type="ProductProperty[]" required>
```

Comment:
It would be nice if this was not a required attribute. Especially when using the "additionalContent" slot you might not want the lists.